crates/state/src/transaction.rs

```diff
@@ -45,12 +45,12 @@ impl Drop for ReaderSpanInfo {
 #[derive(Shrinkwrap)]
 pub struct Reader<'env>(#[shrinkwrap(main_field)] rkv::Reader<'env>, ReaderSpanInfo);
 
-/// If MDB_NOTLS env flag is set, then read-only transactions are threadsafe
+/// If LMDB_NOTLS env flag is set, then read-only transactions are threadsafe
```

Comment:
The flag is actually MDB_NOTLS :slightly_smiling_face: 
I think it's historic from the mdb db before lmdb